src/clinical/clinical-service.ts

```diff
@@ -200,6 +200,15 @@ export const getClinicalData = async (programId: string) => {
 	return data;
 };
 
+const getSingleSpecimenExceptions = async () => {
+	const result = await getExceptions();
+	if (result.success) {
+		return result.data;
+	} else {
+		throw Error('Database failure retrieving single specimen exceptions.');
+	}
```

Comment:
Do you know where this thrown exception gets handled?

Reply:
global top level, similar to the other api calls